docs/conf.py

```diff
@@ -47,10 +47,11 @@
     'docs.css',
 ]
 html_show_sourcelink = True
-if not os.environ.get("READTHEDOCS", False):
-    # The client is needed just for styling some of the sample ad blocks
-    # Ads aren't loaded on these docs outside of RTD
-    html_js_files.append("https://media.ethicalads.io/media/client/beta/ethicalads.min.js")
+
+#if not os.environ.get("READTHEDOCS", False):
+# The client is needed just for styling some of the sample ad blocks
+# Ads aren't loaded on these docs outside of RTD
+html_js_files.append("https://media.ethicalads.io/media/client/alpha/ethicalads.min.js")
```

Comment:
I don't think I'll merge this change but just wanted to force the alpha version of the ad client that supports this adaptive color scheme in the [PR preview](https://ethical-ad-client--229.org.readthedocs.build/en/229/).

Reply:
I have removed this.